backend/src/main/java/com/benine/backend/http/AutoPresetCreationHandler.java

```diff
@@ -42,9 +45,10 @@ public void handle(String s, Request request, HttpServletRequest httpServletRequ
     IPCamera ipcam = (IPCamera) cam;
    
     try {
-      creator.createPresets(ipcam, creator.generateSubViews());
-      respondSuccess(request, httpServletResponse);
-
+      ArrayList<Integer> presetIDs = creator.createPresets(ipcam, creator.generateSubViews());
+      String presetIDList = presetIDs.toString();
+      respond(request, httpServletResponse, presetIDList);
```

Comment:
I think it would be better to have this list of preset IDs in JSON format instead of just the tostring of the arraylist